schemachange/schemachange.h

```diff
@@ -153,21 +153,21 @@ struct schema_change_type {
     int pagesize; /* pagesize override to use */
     int resume;           /* if we are trying to resume a schema change,
                            * usually because there is a new master */
-    bool onstack:1; /* if 1 don't free */
-    bool showsp:1; /* used by plugins */
-    bool must_close_sb:1;
-    bool use_old_blobs_on_rebuild:1;
+    uint8_t onstack:1; /* if 1 don't free */
+    uint8_t showsp:1; /* used by plugins */
+    uint8_t must_close_sb:1;
+    uint8_t use_old_blobs_on_rebuild:1;
 
-    bool nothrevent:1;
-    bool retry_bad_genids:1; /* retrying a schema change (with full rebuild)
+    uint8_t nothrevent:1;
+    uint8_t retry_bad_genids:1; /* retrying a schema change (with full rebuild)
                              because there are old genids in flight */
-    bool dryrun:1;           /* comdb2sc.tsk -y */
-    bool statistics:1;       /* comdb2sc.tsk <dbname> stat <table> */
-    bool use_new_genids:1;   /* rebuilding old genids needs to
+    uint8_t dryrun:1;           /* comdb2sc.tsk -y */
+    uint8_t statistics:1;       /* comdb2sc.tsk <dbname> stat <table> */
+    uint8_t use_new_genids:1;   /* rebuilding old genids needs to
                              get new genids to avoid name collission */
-    bool finalize:1;      /* Whether the schema change should be committed */
-    bool finalize_only:1; /* only commit the schema change */
-    bool fix_tp_badvers:1;
+    uint8_t finalize:1;      /* Whether the schema change should be committed */
+    uint8_t finalize_only:1; /* only commit the schema change */
+    uint8_t fix_tp_badvers:1;
```

Comment:
https://en.cppreference.com/w/c/language/bit_field

Make them all `unsigned`? I see the same `sqliteInt.h` as well (i.e. using `unsigned`).

Reply:
Ok...making them unsigned.